# changes needed to build documentation with Sphinx 0.6
# (thanks to Jakub Wilk <ubanus@users.sf.net> for the patch)
#
# note to backporters: if you will disable this patch, you can downgrade
# python-sphinx minimum requured version to 0.5.1
--- jinja2-2.1.1.orig/docs/conf.py
+++ jinja2-2.1.1/docs/conf.py
@@ -96,7 +96,7 @@
 #html_use_smartypants = True
 
 # use jinja2 for templates
-template_bridge = 'jinjaext.Jinja2Bridge'
+template_bridge = 'sphinx.jinja2glue.BuiltinTemplateLoader'
 
 # no modindex
 html_use_modindex = False
